pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -354,6 +357,8 @@ func (c *Cluster) makeDeployment(osdProps osdProperties, osd OSDInfo) (*apps.Dep
 	} else {
 		deployment.Spec.Template.Spec.InitContainers = append(deployment.Spec.Template.Spec.InitContainers, c.getPVCInitContainer(osdProps.pvc))
 		k8sutil.AddLabelToDeployement(OSDOverPVCLabelKey, osdProps.pvc.ClaimName, deployment)
+		k8sutil.AddLabelToDeployement(failureDomainKey, failureDomainValue, deployment)
```

Comment:
Let's always add these labels to the deployment and pod. If not using a pvc, we would just expect `portable` to be false and the failureDomain to be the host name.

Reply:
done